web/src/e2e/index.e2e.test.tsx

```diff
@@ -582,6 +582,46 @@ describe('e2e test suite', () => {
                     await driver.assertWindowLocation(repoBaseURL + navigationTest.symbolPath, true)
                 })
             }
+
+            const highlightSymbolTests = [
+                {
+                    name: 'highlights correct line for Go',
+                    filePath:
+                        '/github.com/sourcegraph/go-diff@3f415a150aec0685cb81b73cc201e762e075006d/-/blob/diff/diff.go',
+                },
+                {
+                    name: 'highlights correct line for Typescript',
+                    filePath:
+                        '/github.com/sourcegraph/sourcegraph-typescript@a7b7a61e31af76dad3543adec359fa68737a58a1/-/blob/server/src/cancellation.ts',
+                },
+            ]
+
+            for (const highlightSymbolTest of highlightSymbolTests) {
```

Comment:
```suggestion
            for (const { name, filePath } of highlightSymbolTests) {
```

(minor) this is a bit terser.